sync/sync-impl/src/main/java/com/duckduckgo/sync/impl/SyncRepository.kt

```diff
@@ -34,9 +34,10 @@ interface SyncRepository {
     fun getAccountInfo(): AccountInfo
     fun storeRecoveryCode()
     fun removeAccount()
-    fun logout(): Result<Boolean>
+    fun logout(deviceId: String = ""): Result<Boolean>
```

Comment:
I would remove the default value for `deviceId`. 
```suggestion
    fun logout(deviceId: String): Result<Boolean>
```